llama_hub/imdb_review/scraper.py

```diff
@@ -225,7 +226,7 @@ def main_scraper(
 
     print(f"Number of reviews scraped: {len(reviews_date)}")
     if generate_csv:
-        os.makedirs("movie_reviews", exist_ok=True)
+        os.makedirs(reviews_folder, exist_ok=True)
```

Comment:
do we need to make this change in this PR? if there's imdb changes let's make that in a sep PR

Reply:
I contributed imdb_scraper sometime back, but this was just a minor change that does not change much. For making it a separate PR, do I have to undo the changes and commit again, or is there any other way? Sorry if this is a stupid question